### PR TITLE
Ticket06/ref reset button

### PR DESCRIPTION
**Pull request by claramiralles**
Can you please review @patrozada?  The changes are clean code and arrow functions. Let me know if there is some extra info you need!

### CODE REVIEW

**Comment by claramiralles** [comment]
@valerialp can you review the new changes? They are: share function changed to arrow function in partial share.js. Let me know if you need any extra information. Thanks!

File: _src/assets/js/reset-button.js [hunk 1]
```diff
@@ -1,96 +1,50 @@
...
+};
+const resetSN = () => {
+  const arrSN = document.querySelectorAll('.card__item-list');
+  for (let i = 0; i < arrSN.length; i++) {
+    arrSN[i].classList.remove('card__item-list');
+    arrSN[i].classList.add('card__item-list-no-border');
+  }
+};
+const resetColor = () => {
+  selectEl.checked = true;
```
Comment (valerialp) [inline _src/assets/js/reset-button.js:35]: this function must add the default color

Comment (claramiralles) [reply _src/assets/js/reset-button.js:35]: Thanks Valeria, I just added it: "included add and remove of classlists in reset color function in partial reset-button.js"

Comment (valerialp) [reply _src/assets/js/reset-button.js:35]: Great job!! <3

**Review by valerialp — APPROVED** [review]

**Pull request merged**